src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -1,462 +1,407 @@
         }
-    }
-#endif
 
-    [Fact]
-    [PlatformSpecific(PlatformID.Windows)] // max directory length not fixed on Unix
-    public static void CreateDirectory_DirectoryEqualToMaxDirectory_CanBeCreated()
-    {   // Recursively creates directories right up to the maximum directory length ("247 chars not including null")
-        using (TemporaryDirectory directory = new TemporaryDirectory())
+        [Fact]
+        [PlatformSpecific(PlatformID.Windows)] // testing drive labels
+        [ActiveIssue(1221)]
+        public void SubdirectoryOnNotReadyDriveAsPath_ThrowsIOException()
         {
-            PathInfo path = IOServices.GetPath(directory.Path, IOInputs.MaxDirectory, IOInputs.MaxComponent);
-
-            // First create 'C:\AAA...AA', followed by 'C:\AAA...AAA\AAA...AAA', etc
-            foreach (string subpath in path.SubPaths)
+            var drive = IOServices.GetNotReadyDrive();
+            if (drive == null)
             {
-                DirectoryInfo result = Directory.CreateDirectory(subpath);
-
-                Assert.Equal(subpath, result.FullName);
-                Assert.True(Directory.Exists(result.FullName));
+                Console.WriteLine("Skipping test. Unable to find a not-ready drive, such as CD-Rom with no disc inserted.");
+                return;
             }
+
+            // 'Device is not ready'
+            Assert.Throws<IOException>(() =>
+            {
+                Create(Path.Combine(drive, "Subdirectory"));
+            });
         }
-    }
 
-    [Fact]
-    [PlatformSpecific(PlatformID.Windows)] // max directory length not fixed on Unix
-    public static void CreateDirectory_DirectoryEqualToMaxDirectory_CanBeCreatedAllAtOnce()
-    {   // Creates directories up to the maximum directory length all at once
-        using (TemporaryDirectory directory = new TemporaryDirectory())
+        [Fact]
+        [ActiveIssue(1221)]
+        [PlatformSpecific(PlatformID.Windows)] // testing drive labels
+        public void NonExistentDriveAsPath_ThrowsDirectoryNotFoundException()
         {
-            PathInfo path = IOServices.GetPath(directory.Path, IOInputs.MaxDirectory, maxComponent: 10);
-
-            DirectoryInfo result = Directory.CreateDirectory(path.FullPath);
+            var drive = IOServices.GetNonExistentDrive();
+            if (drive == null)
+            {
+                Console.WriteLine("Skipping test. Unable to find a non-existent drive.");
+                return;
+            }
 
-            Assert.Equal(path.FullPath, result.FullName);
-            Assert.True(Directory.Exists(result.FullName));
+            Assert.Throws<DirectoryNotFoundException>(() =>
+            {
+                Create(drive);
+            });
         }
-    }
 
-    [Fact]
-    public static void CreateDirectory_ValidPathWithTrailingSlash_CanBeCreated()
-    {
-        using (TemporaryDirectory directory = new TemporaryDirectory())
+        [Fact]
+        [ActiveIssue(1221)]
```

Comment:
Why is the test for a non-existent drive an active issue?  I understand the problem with mounting drives, but this seems to not need to mount and instead just wants to make sure things work in the opposite case.